### ISSUE TITLE
Gremlin Server 3.1.0 breaks the tests.

### ISSUE THREAD

**Issue by smolinari** [spider/spider#105]
I was playing with my new fangled [vagrant box](https://github.com/SkooppaOS/spider-box) and thought I'd try out the newer server versions of Neo4j 2.3.1 and Gremlin 3.1.0. Neo4j is ok, but Gremlin  3.1.0 breaks the tests. I reverted back to 3.0.2, because it is now so easy to do, and everything ran with no problems. 

Scott

**Comment by dmill-bz** [comment]
yes 3.1.0 requires a different set of configuration files. Once I make the changes to the travis build in gremlin-php it will be easier to update spider.
